src/main/java/net/redstoneore/legacyfactions/integration/vault/VaultEngine.java

```diff
@@ -194,18 +194,18 @@ public static boolean transferMoney(EconomyParticipator invoker, EconomyParticip
         }
 
         // Transfer money
-        EconomyResponse erw = econ.withdrawPlayer(fromAcc, amount);
+        EconomyResponse erw = econ.withdrawPlayer(fromAcc.getName(), amount);
```

Comment:
I know its not your fault, but the previous maintainer used a lot of acronyms which we're trying to move away from - as these can make it difficult to read. 

For example, `erw` should be `ecomomyResponseWithdraw`